src/components/network/network-table.js

```diff
@@ -69,21 +80,28 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
+const ROW_HEIGHT = 48;
+
 const NetworkTable = (props) => {
     const classes = useStyles();
 
+    const notifiedDisplayedColumns = useSelector(
+        (state) => state.displayedColumns
+    );
+
     const [tabIndex, setTabIndex] = useState(0);
     const [lineEdit, setLineEdit] = useState({});
     const [rowFilter, setRowFilter] = useState(undefined);
     const [popupSelectColumnNames, setPopupSelectColumnNames] = useState(false);
+    const [selectedColumnsNames, setSelectedColumnsNames] = useState(null);
```

Comment:
not sure about duplicating the values in redux here in the state of the component..

Reply:
We need a local state which immediately reflects the choices made in the selection dialog (popuup)